examples/.config/model_params_pytorch.json

```diff
@@ -18,6 +18,96 @@
       "batch_size": 100,
       "new_benchmark": false
     },
+    "googlenet_fx": {
```

Comment:
Adding a new strategy should not change the file under `examples`.
Please keep all of the files under `examples` be same as the master.

Reply:
resolved: 
909549b1062f69a74b38812f03f025832e20cd21
f47c016308e95fa507539620f4d84c9d189e8e116